<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AntennaDataViewerClass</class>
 <widget class="QMainWindow" name="AntennaDataViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>625</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>625</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>625</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AntennaDataViewer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ant_logo.jpg</normaloff>ant_logo.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="PlotWidget" native="true">
    <property name="geometry">
     <rect>
      <x>429</x>
      <y>10</y>
      <width>581</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblSelect">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ограничения на выборку из БД</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblParInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>145</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Параметры, чье влияние исследуем</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblParOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>335</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Параметры, влияние на которых исследуем</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="listParInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>261</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listParOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>261</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="butCorr">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>520</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Рассчитать корреляцию</string>
    </property>
   </widget>
   <widget class="QLabel" name="lboCorrLine">
    <property name="geometry">
     <rect>
      <x>291</x>
      <y>180</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Линейная корреляция</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leCorrLine">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leCorrLineA">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lboCorrLine_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Линейная регрессия
(y = a + bx + eps)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lboCorrLine_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>a=</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbLine">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="listDBSelect">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>30</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leCorrLineB">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lboCorrLine_13">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>b=</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leCorrLineEPS">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lboCorrLine_14">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>130</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>eps=</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leCorrLineDlt">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lboCorrLine_15">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Δ=</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lboCorrLine_7">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Параболическая
регрессия
(y = a + bx + сxx+ eps)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbPar">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>270</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="leCorrParB">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>350</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lboCorrLine_16">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>480</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Параболическая
корреляция</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leCorrParA">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>320</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leCorrParDlt">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>440</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leCorrPar">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>510</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lboCorrLine_17">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>b=</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lboCorrLine_18">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>a=</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lboCorrLine_19">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>440</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Δ=</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lboCorrLine_20">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>410</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>eps=</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leCorrParEPS">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>410</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lboCorrLine_21">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>380</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>b=</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leCorrParC">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>380</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>База Данных</string>
    </property>
    <addaction name="actChangeDB"/>
    <addaction name="actCopyDB"/>
    <addaction name="actDelDataDB"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Программа Крона</string>
    </property>
    <addaction name="actOpenAntKrona"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actChangeDB">
   <property name="text">
    <string>Сменить БД</string>
   </property>
  </action>
  <action name="actCopyDB">
   <property name="text">
    <string>Копировать БД</string>
   </property>
  </action>
  <action name="actDelDataDB">
   <property name="text">
    <string>Удалить данные из БД</string>
   </property>
  </action>
  <action name="actOpenAntKrona">
   <property name="text">
    <string>Открыть антенну в Кроне</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="antennadataviewer.qrc"/>
 </resources>
 <connections/>
</ui>
